### PR TITLE
MAISTRA-1844: Update operator to support maistra 1.1.9

### PR DESCRIPTION
**Pull request by knrc**

### CODE REVIEW

**Review by rcernich — APPROVED** [review]

**Comment by knrc** [comment]
This should be good to go once #557 is merged

File: resources/helm/v1.1/istio/charts/kiali/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 description: Kiali is an open source project for service mesh observability, refer to https://www.kiali.io for details.
 name: kiali
-version: 1.9.1
```
Comment (dgn) [inline resources/helm/v1.1/istio/charts/kiali/Chart.yaml]: was this change intentional? it doesn't look like anything but the version number changed here

Comment (knrc) [reply resources/helm/v1.1/istio/charts/kiali/Chart.yaml]: This change was pulled in by the Makefile, let me check what is going on

Comment (knrc) [reply resources/helm/v1.1/istio/charts/kiali/Chart.yaml]: cleaning everything out and rerunning the Makefile fixes this, I'll push up a new version now

**Review by dgn — APPROVED** [review]

**Pull request merged**